apps/libsignpost/signpost_energy_monitors.c

```diff
@@ -0,0 +1,186 @@
+#include "i2c_selector.h"
+#include "ltc294x.h"
+#include "signpost_energy_monitors.h"
+#include "max17205.h"
+
+static uint8_t module_num_to_selector_mask[8] = {0x4, 0x8, 0x10, 0, 0, 0x20, 0x40, 0x80};
+static ltc294x_model_e model;
+static uint16_t prescaler;
+
+
+////////////////////////////////////////////////////////////
+//Initialization functions
+//Make sure to call the right one
+/////////////////////////////////////////////////////////
+void signpost_energy_init_ltc2941 (void) {
```

Comment:
There needs to be an i2c eeprom driver written - which feels like something that should go in another PR